Documentation/ceph-cluster-crd.md

```diff
@@ -391,7 +391,15 @@ spec:
   * This format is required in order to use the NetworkAttachmentDefinition across namespaces.
   * In Openshift, to use a NetworkAttachmentDefinition (NAD) across namespaces, the NAD must be deployed in the `default` namespace. The NAD is then referenced with the namespace: `default/rook-public-nw`
 
-#### Known issues with multus
+#### Known limitations with Multus
+
+Daemons leveraging Kubernetes service IPs (Monitors, Managers, Rados Gateways) are not listening on the NAD specified in the `selectors`.
```

Comment:
To be clear, are the following statements true?
- No incoming traffic to the mons, mgr, and rgw can be received over the multus network. Other network bindings must be used to target mons (for external clusters) and rgw. 
- A rook cluster using multus cannot be used as the target for an external cluster.

Reply:
It's also a little unclear to me what the effects of this limitation are. I think that info would be good for users to know.